src/main/resources/templates/showProduct.html

```diff
@@ -123,7 +123,7 @@ <h5 class="card-title" th:text="|Leave a Review for ${productName}|"></h5>
                         <textarea class="form-control" id="body" rows="5" placeholder="Write down your thoughts" th:field="*{body}" required></textarea>
                     </div>
 
-                    <button type="submit" class="btn btn-primary" id="formSubmit">Submit</button>
+                    <button type="submit" class="btn btn-primary" id="formSubmit" th:text="${alreadyPostedReview ? 'Edit' : 'Submit'}"></button>
```

Comment:
I feel like instead of "Edit" it should be "Edit review" or "Update Review" so it's more intuitive

Reply:
It's only Edit, so the size is similar to the size of the Submit button.

Reply:
Yea I don't think that's too big of a deal